modules/enabling-the-multi-model-serving-platform.adoc

```diff
@@ -14,6 +14,7 @@ endif::[]
 ifdef::upstream[]
 * If you are using specialized {productname-short} groups, you are part of the admin group (for example, {odh-admin-group}) in OpenShift.
 endif::[]
+* Your cluster administrator has _not_ edited the {productname-short} dashboard configuration to disable the ability to select ModelMesh as a serving platform. For more information, see link:{rhoaidocshome}/html/managing_resources/customizing-the-dashboard#ref-dashboard-configuration-options_dashboard[Dashboard configuration options].
```

Comment:
We shouldn't refer to ModelMesh as a serving platform. ModelMesh is the _component_ that underpins the multi-model serving platform. 

In that case, we should say "...to disable the ability to select the multi-model serving platform, which uses the ModelMesh component."

I know this description came from [here](https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2-latest/html/managing_resources/customizing-the-dashboard#ref-dashboard-configuration-options_dashboard), so we need to get it updated there too. In addition, that description should not use "Serving Platform", capitalized.

Reply:
Addressed in latest commit.